src/lib.rs

```diff
@@ -159,13 +159,18 @@ pub trait MiniscriptKey: Clone + Eq + Ord + fmt::Debug + fmt::Display + hash::Ha
     /// Used in the sha256 fragment
     type Sha256: Clone + Eq + Ord + fmt::Display + fmt::Debug + hash::Hash;
 
+    /// The associated [`hash256::Hash`] type for this [`MiniscriptKey`] type.
+    /// Used in the sha256 fragment
```

Comment:
```suggestion
    /// The associated [`hash256::Hash`] for this `MiniscriptKey`, used in the hash256 fragment.
```

If this form is agreeably, then could use it for the comment on `Sha256` also.

Reply:
Done